spec/models/food_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
```

Comment:
- [ ] Kindly make sure to implement integration tests for your controllers as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=Write%20integration%20tests%20with%20capybara%20gem) requirement.

Reply:
@ShadyShawkat Thanks this command runs the intergration tests `rspec test/integration`